src/Host/Client/Impl/Host/LocalRHostConnector.cs

```diff
@@ -93,6 +93,8 @@ public LocalRHostConnector(string name, string rHome, string rhostDirectory = nu
                         FileName = rhostBrokerExe,
                         UseShellExecute = false,
                         Arguments =
+                            $" --logging:logHostOutput true" +
```

Comment:
Ok

---
In reply to: [75219094](https://github.com/Microsoft/RTVS/pull/2178#discussion_r75219094) [](ancestors = 75219094)